TABScrollingContentViewTests/TABScrollingContentViewTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  TABScrollingContentViewTests.swift
+//  TABScrollingContentViewTests
+//
+//  Created by Kane Cheshire on 20/02/2017.
+//  Copyright © 2017 The App Business. All rights reserved.
+//
+
+import XCTest
+@testable import TABScrollingContentView
+
+class TABScrollingContentViewTests: XCTestCase {
+  
+  var scrollingContentView: VerticalScrollingContentView?
```

Comment:
I'd go with an implicitly unwrapped option for your system under test, it'll clean up all the optionals in the test